uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -361,10 +361,12 @@ define([
         if (missingRequiredFields.length) {
           var notification = CRM.alert(missingRequiredFields.join(', '),
             missingRequiredFields.length === 1 ? 'Required field' : 'Required fields', 'error');
+
           $timeout(function () {
-            notification.close();
+            notification && notification.close();
```

Comment:
Every change needs to be documented. I am not sure what this change is about. Add details about it in the description.